packages/splitio-adapter/modules/adapter/adapter.js

```diff
@@ -95,11 +96,13 @@ const configure = ({
   user,
   onFlagsStateChange,
   onStatusStateChange,
+  ...adapterArgs
```

Comment:
Not sure about the spread. Should be just have an `options` key we expect to receive here? That way the adapter remains a bit more extensible and we don't "by accident" spread in too many things.

Reply:
That's what I meant by nesting this in a key.
You'd prefer ano object like this: 
```
adapterArgs={{
  authorizationKey,
  user,
  options: {
    core: {
      labelsEnabled: false,
    },
  },
}}
```

Reply:
Thanks for the feedback. Not each adapter will however have a key. Things like localstorage or custom http adapters. That‘s why I decided against it. For performance, no matter how deep an object or array it will we a new reference on render. At work I have a memoized function for creating the args which upon equal args returns and referentially equivilant object for the adapterArgs.